backend/muster-roll-service/src/main/java/org/egov/web/models/AttendanceEntry.java

```diff
@@ -21,6 +22,10 @@
 @NoArgsConstructor
 @Builder
 public class AttendanceEntry {
+
+    @JsonProperty("id")
+    private UUID id = null;

```

Comment:
Change all the UUID types to String

Reply:
As per discussion changed all id variables as String and timestamp as BigDecimal.